services/memory.go

```diff
@@ -0,0 +1,39 @@
+package services
+
+import (
+	"github.com/AndresCravioto/academy-go-q42021/loader"
+	"log"
+	"os"
+)
+
+type Stands struct {
+	Store *[]*loader.StandData `json:"store"`
+}
+
+func (b *Stands) Initialize() {
+	filename := "./assets/stands.csv"
```

Comment:
You would like to handle the file path on a way that is not hardcoded, we want our app to be flexible enough to work with any file, specially for unit testing